apps/frontend/src/modules/courses/slices/actions.ts

```diff
@@ -1,22 +1,24 @@
 import { createAsyncThunk } from "@reduxjs/toolkit";
 
+import { NotificationMessage } from "~/libs/modules/notification/notification.js";
 import { type AsyncThunkConfig } from "~/libs/types/types.js";
 
 import {
 	type CourseDto,
 	type CourseSearchFilterDto,
 	type CourseSearchGetAllResponseDto,
+	type CourseUpdateRequestDto,
 } from "../libs/types/types.js";
 import { name as sliceName } from "./courses.slice.js";
 
-const getAll = createAsyncThunk<
+const getAllByFilter = createAsyncThunk<
```

Comment:
```suggestion
const getAllByVendor = createAsyncThunk<
```
I think it would be better. change names (reducer key, action name, etc)